perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -654,7 +654,7 @@ sub response_templates {
 #     Note:   this only makes sense when called on a problem that has been sent!
 sub can_display_external_id {
     my $self = shift;
-    if ($self->external_id && $self->send_method_used && $self->bodies_str =~ /2237/) {
+    if ($self->external_id && $self->send_method_used && $self->bodies_str =~ /(2237|2550)/) {
```

Comment:
This is the only hard-coded ID (now IDs) in Problem.pm; can we think of a way to factor it out to the cobrand while we're here? I guess passing `$c` through would be most straightforward, and one way that's already done around here (for passing down to body's URL? :-/ ).